jenkinsTestServicedBuildDeps.sh

```diff
@@ -17,7 +17,7 @@ export PACKER_CACHE_DIR="${HOME}/packer_cache"
 export PACKER_LOG=1
 export PACKER_LOG_PATH="${BUILD_DIR}/packer-${SERVICED_RPM}.log"
 
-SERVICED_ARTIFACT_BASENAME=serviced-1.5.0-0.0.223.unstable-unstable-centos7.2.1511-bld-28
+SERVICED_ARTIFACT_BASENAME=serviced-1.6.5-0.0.333.unstable-unstable-centos7.2.1511-bld-39
```

Comment:
serviced 1.6.5 is unreleased version.  Perhaps a 1.6.4 build would be preferable....?